### ISSUE TITLE
RFC: Tent names as the vertical header in block view

### ISSUE THREAD

**Issue by maks** [Wilm0r/giggity#8]
While 20a6ff50df1b2ad06cbeb9eb4c63175149453690 was a quick fix, I think it may be worthwhile to have  the option of switching the axis around in the block view to present the tent names as a header along the top with time running downwards, eg the way the [LCA schedule is formatted on their website](https://linux.conf.au/schedule/)

I think this would also give better spacing for the titles of individual events to be displayed more clearly.

**Comment by Wilm0r** [comment]
Yeah I've been thinking of this for a while already. It'd work especially well for schedules with few rooms. Probably not something I'll be able to implement in time for LCA next week though but may give it a shot.

**Comment by maks** [comment]
Thanks for the quick feedback!
Yes I'm presses for time also but may have a chance to work on it during LCA if you dont get to it before then.

**Comment by Wilm0r** [comment]
:-D

PR definitely welcome on that. I've not done it also because pushing a somewhat major change right before an event just seems a little risky.

With the vertical/transposed layout I'm also thinking of, where space allows, automatically scaling to fill the screen horizontally (though the LCA schedule has too many rooms for that already), and maybe putting more info in the boxes (actual times, subtitles?).

I'd also love to style that view a little more, it's pretty simple ATM, really that view is the oldest piece of code in Giggity. Only improvement I've made recently is to replace the H+V combined scrollview with something a little more native (that supports stuff like flinging). But it's still built with an AbsoluteLayout which was deprecated pretty much in API level 1 already. :-D But IMHO it's really the best approach to this specific problem.

**Comment by waldyrious** [comment]
I came to the repo to ask for precisely this, and nearly missed this issue because the title is confusing ("tent names" only applies to certain conferences — others may use rooms, buildings, etc.

I would propose a more generic title to make this issue easier to find for others, e.g. "Transposed block view (time running downwards)".

**Comment by Wilm0r** [comment]
<img width="360" height="780" alt="Image" src="https://github.com/user-attachments/assets/e024b6f6-9b39-43c0-a5b3-d354c68a6f61" />

Example of the last idea. I do think this is better yeah.
